posthog/api/event.py

```diff
@@ -123,10 +123,29 @@ def _prefetch_events(self, events: List[Event]) -> List[Event]:
         return events
 
     def list(self, request: request.Request, *args: Any, **kwargs: Any) -> response.Response:
-        events = [event for event in self.get_queryset()[0: 101]]
-        prefetched_events = self._prefetch_events(events[0:100])
+        queryset = self.get_queryset()
+        monday = now() + timedelta(days=-now().weekday())
+        events = queryset.filter(timestamp__gte=monday.replace(hour=0, minute=0, second=0))[0: 101]
+
+        if len(events) < 101:
+            events = queryset[0: 101]
+
+        prefetched_events = self._prefetch_events([event for event in events])
+        path = request.get_full_path()
+
+        reverse = request.GET.get('orderBy', '-timestamp') != '-timestamp'
+        if len(events) > 100:
+            next_url: Union[bool, str] = '{}{}{}={}'.format(
+                path,
+                '&' if '?' in path else '?',
+                'after' if reverse else 'before',
+                events[99].timestamp.strftime('%Y-%m-%dT%H:%M:%S.%fZ')
+            )
+        else:
+            next_url = False
+
         return response.Response({
-            'next': len(events) > 100,
+            'next': next_url,
```

Comment:
Did you mean to change how `eventtablelogic` works? This URL looks good but the table logic doesn't use it yet because before it was just a boolean

Reply:
It wasn't part of this ticket so I'll get this in and come back to that.